napari/layers/image/_constants.py

```diff
@@ -36,6 +36,9 @@ class Rendering(StringEnum):
               the result is opaque.
             * mip: maxiumum intensity projection. Cast a ray and display the
               maximum value that was encountered.
+            * attenuated_mip: attenuated maxiumum intensity projection. Cast a
+              ray and attenuate values based on integral of encountered values,
+              display the maximum value that was encountered after attenuation.
```

Comment:
would it be appropriate to add a "qualitative hint" here, like "(makes nearer objects appear more prominent)" or something like that?

Reply:
done